apigw-websocket-mapping-template-authorizer/src/sendmessage.mjs

```diff
@@ -0,0 +1,45 @@
+        import { DynamoDBClient, ScanCommand, DeleteItemCommand } from "@aws-sdk/client-dynamodb";
+        import { ApiGatewayManagementApiClient, PostToConnectionCommand } from "@aws-sdk/client-apigatewaymanagementapi";
+        const ddbClient = new DynamoDBClient({ region: process.env.AWS_REGION });
+
+        export const handler = async (event) => {
+            let connectionData;
+            console.log("event received:", event); //check the event received by Lambda
+
+            //scanning DB table to get the Connection ID
+            const scanParams = {
+              TableName: process.env.TABLE_NAME,
+              ProjectionExpression: "connectionId",
+            };
+            
+            const scanCommand = new ScanCommand(scanParams);
+            const responseDynamo = await ddbClient.send(scanCommand);
+            connectionData = responseDynamo.Items;
+            const connectionId = connectionData[0].connectionId.S;
+            console.log("connectionData:", connectionData); // print info about the DB connection
+            
+            //building endpoint from env variables, can also be buit from request parameters 
+            const endpoint = "https://" + process.env.API_ID + ".execute-api." + process.env.AWS_REGION + ".amazonaws.com/" + process.env.STAGE + "/";
+            
+            const apigwManagementApi = new ApiGatewayManagementApiClient({ apiVersion: "2018-11-29",
+            endpoint: endpoint,
+            });
+            
+            const body = JSON.parse(event.body);
```

Comment:
CWE-502,1321 - Deserialization of untrusted object High
It appears that you are using JSON.parse() on tainted user input. This could lead to data compromise, acccess control attacks, prototype pollution, denial of service (DoS) attacks, or authentication bypass. To increase the security of your code, do not parse tainted user input.